tests/e2e/tests/critical-path/tree-view/tree-view-improvements.e2e.ts

```diff
@@ -0,0 +1,169 @@
+import { Selector, t } from 'testcafe';
+import { acceptLicenseTermsAndAddDatabaseApi } from '../../../helpers/database';
+import {
+    BrowserPage, CliPage
+} from '../../../pageObjects';
+import {
+    commonUrl,
+    ossStandaloneBigConfig,
+    ossStandaloneConfig
+} from '../../../helpers/conf';
+import { KeyTypesTexts, rte } from '../../../helpers/constants';
+import { deleteStandaloneDatabaseApi } from '../../../helpers/api/api-database';
+import { Common } from '../../../helpers/common';
+import { verifyKeysDisplayedInTheList, verifyKeysNotDisplayedInTheList } from '../../../helpers/keys';
+
+const browserPage = new BrowserPage();
+const common = new Common();
+const cliPage = new CliPage();
+let keyNames: string[];
+let keyName1: string;
+let keyName2: string;
+let keynameSingle: string;
+let index: string;
+
+fixture`Tree view navigations improvement tests`
+    .meta({ type: 'critical_path', rte: rte.standalone })
+    .page(commonUrl);
+test
+    .before(async () => {
+        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+    })
+    .after(async () => {
+        await t.click(browserPage.patternModeBtn);
+        await browserPage.deleteKeysByNames(keyNames);
+        await deleteStandaloneDatabaseApi(ossStandaloneConfig);
+    })('Tree view preselected folder, Filterred Tree view preselected folder, Refreshed Tree view preselected folder, Search capability Filterred Tree view preselected folder', async t => {
```

Comment:
too long name
you can name it just 'Tree view preselected folder'

and below before each ".expect/verify" in test add comment with test from checklist (to be understandable what each verification step actually checks)